File: app/src/main/java/collection/ex02.java
package collection;

import java.util.HashMap;
import java.util.Map;

public class ex02 {
    public static void main(String[] args) {
        Map<String, String> dict = new HashMap<>();
        dict.put("Network", "network");
        dict.put("Thread", "thread");

        for (String key : dict.keySet()) {
            System.out.println(key + " : " + dict.get(key));
        }
    }
}

File: app-kotlin/src/main/kotlin/collection/ex02.kt
package collection

class ex02 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val dict = mutableMapOf(
                "Network" to "network",
                "Thread" to "thread"
            )

            for ((key, value) in dict) {
                println("$key : $value")
            }
        }
    }
}
